seonshine_frontend/src/utils/persistCache/index.ts

```diff
@@ -1,14 +1,19 @@
 class PersistCache {
-  static save(key: string, value: string): void {
-    localStorage.setItem(key, value);
+  static save(key: string, value: string, rememberMe?: boolean) {
+    if (rememberMe) {
+      localStorage.setItem(key, value);
+    } else {
+      sessionStorage.setItem(key, value);
+    }
   }
 
   static read(key: string): string | null {
-    return localStorage.getItem(key);
+    return localStorage.getItem(key) || sessionStorage.getItem(key);
```

Comment:
pleae remove sessionStorage.getItem(key). `PersistCache` for permanent saving data, not apply for sessionStorage